charts/k8s/values.yaml

```diff
@@ -403,6 +403,11 @@ isolation:
 
   podSecurityStandard: baseline
 
+  # If enabled will add node/proxy permission to the cluster role
```

Comment:
You will have to do this in values.yaml file of the other 3 charts as well.

Reply:
Whoa... Thanks!